testsuite/factories.py

```diff
@@ -110,6 +110,37 @@ class Meta:
                    'manage_media_participant', 'view_participant')
 
 
+class RoleUserMeetingFactory(SQLAlchemyModelFactory):
+
+    class Meta:
+        model = models.RoleUser
+        sqlalchemy_session = models.db.session
+
+    role = SubFactory(RoleFactory)
+    user = SubFactory(UserFactory)
+    meeting = SubFactory(MeetingFactory)
+
+    @post_generation
+    def staff(self, create, extracted, **kwargs):
+        if not create:
+            return
+        staff = extracted or StaffFactory(user=self.user)
+        return staff
+
+
+class JobsFactory(SQLAlchemyModelFactory):
+    class Meta:
+        model = models.Job
+        sqlalchemy_session = models.db.session
+
```

Comment:
You can auto-generate the `id` using `Sequence` or `sequence`. You have some examples in this file.